starters/blog/package.json

```diff
@@ -30,8 +30,7 @@
     "react-dom": "^17.0.1",
     "react-helmet": "^6.1.0",
     "typeface-merriweather": "0.0.72",
-    "typeface-montserrat": "0.0.75",
-    "webpack": "^5.35.0"
```

Comment:
I added this because some of the gatsby plugins requires `webpack` to be provided by the parent, removing it will make them unmet peer dependencies and be rejected by PnP again

Reply:
Interesting cause we don't want that people install webpack, i'll see why it's necessary here.